src/components/useForm.js

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { makeStyles } from "@material-ui/core";
+
+export function useForm(initialFValues, validateOnChange = false) {
+  const [values, setValues] = useState(initialFValues);
+
+  const handleInputChange = (e) => {
```

Comment:
callback this function.  
https://react.dev/reference/react/useCallback#usage  
  
As a practice, all values returned from hooks should be memoized/callbacked if function.  
The consumers of this hooks might use it for useEffects etc...